src/Fetch.js

```diff
@@ -7,8 +7,9 @@ export default class Fetch extends Component {
     }
 
     componentDidMount = async () => {
-        const response = await fetch.get('http://futuramaapi.herokuapp.com/api/quotes');
-        this.setState({ quotes: response.body });
+        const response = await fetch.get('https://alchemy-pokedex.herokuapp.com/api/pokedex');
```

Comment:
nice work! I left a comment in Canvas, but you will be using different fetch calls to sort and filter your results, but instead of using methods, only using the API